test/onnx/test_operators.py

```diff
@@ -549,6 +549,14 @@ def test_nonzero(self):
         x = torch.tensor([[[2., 2.], [1., 0.]], [[0., 0.], [1., 1.]]], requires_grad=True)
         self.assertONNX(lambda x: torch.nonzero(x), x)
 
+    def test_topk(self):
+        x = torch.arange(1., 6., requires_grad=True)
+        self.assertONNX(lambda x: torch.topk(x, 3), x)
+
+    def test_topk_opset10(self):
```

Comment:
this is a duplicate with test_master_opset, merge two?

Reply:
The plan was to refactor the tests in a second PR.
We would split this file to have one for opset9 tests and another for opset10 tests (so we could have multiple tests for opset 10 as well, and we will move both these tests to the opset10 file).
 What is your thought about this?

Reply:
I am hesitate to add too many tests rely on expect files. I am thinking to have a new check mechanism, which doesn't rely on expect files.

We can do the following:
1) check the key components in onnx proto, such as ir_version, opset
2) check the target type of node exists in the graph, probably also check the attribute value
3) rely on the onnx checker to do the schema check

Reply:
This is possible for operators that have similar schemas for the different opsets. In this case, topk has a different schema between opset 9 and 10. Would we want to have an expect file for each, or just one for one of the opsets?

Reply:
I agree that we should reduce the reliance on expect-file based tests. We can discuss how best to do that. For this PR we are incrementally making changes to enable opset 10 dispatch.